plugins/double_sided_cubes.js

```diff
@@ -0,0 +1,45 @@
+(()=> {
+	Plugin.register('double_sided_cubes', {
+		title: 'Double Sided Cubes',
+		author: 'SnaveSutit',
+		description: 'Creates inverted duplicates of the selected cube(s) to allow double-sided rendering in java edition.',
+		icon: 'flip_to_back',
+		version: '1.0.0',
+		variant: 'both',
+		onload() {
+			cube_action = new Action({
```

Comment:
Declare a variable with one of the variable keywords (var, let, const)

In this case, you should declare it above `Plugin.register` so that it's available in the `onunload` event as well.